FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,34 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int8.h>
+#include <Arduino.h>
+std_msgs::Int8 fuel_level_percent_full;
+ros::Publisher pub_fuel_level("fuel_level_percent_full", &fuel_level_percent_full);
+ros::NodeHandle nh;
+
+const long LOOP_DELAY = 200;
+const int READ_PIN = A7;
+
+void setup()
+{
+    nh.initNode();
+    Serial.begin(9600);
+    nh.getHardware()->setBaud(9600);
+    nh.advertise(pub_fuel_level);
+}
+
+void loop() {
+    static unsigned long current_time = millis();
+    static unsigned long last_fuel_level_pub_time = current_time;
+    current_time = millis();
```

Comment:
You don't need this if you have line 21

Reply:
I think that needs to be there to re-update it every loop, right?

Reply:
Fixed this